bin/terraform_dsl

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
+$LOAD_PATH.unshift(File.expand_path('../lib', __dir__))
+require 'terraform_dsl/cli'
+
+begin
+  ENV["THOR_DEBUG"] = "1"
+  Terraform::CLI.start(ARGV)
+rescue Thor::UndefinedCommandError, Thor::UnknownArgumentError, Thor::AmbiguousCommandError, Thor::InvocationError => e
+  $stderr.puts(e.message)
+  exit(64)
+rescue Thor::Error => e
+  $stderr.puts(e.message)
+  exit(1)
+end
```

Comment:
Normally thor always exits with exit status 0, which is shitty when building scripts. The debug options allows you to handle exceptions yourself.